tests/test_plotting_api.py

```diff
@@ -19,31 +19,58 @@
 from jaxley.synapses import IonotropicSynapse
 
 
-def test_cell():
-    dirname = os.path.dirname(__file__)
-    fname = os.path.join(dirname, "swc_files", "morph.swc")
-    cell = jx.read_swc(fname, nseg=4)
+@pytest.fixture(scope="module")
```

Comment:
This is awesome! I think we could even move them to a separate file, to make them available to all other tests. `tests/fixtures.py`.

Reply:
Feel free to replace as many of the other tests that rely on `comp` or `net` etc. with these fixtures. But also fine if you don't and then just leave #449 open and just add a quick comment to it.